Makefile

```diff
@@ -6,7 +6,7 @@ include subtrees/z_quantum_actions/Makefile
 # This project has a lower-than-average test coverage. Let's temporarily
 # override the required coverage threshold to let us CI test runs finish, add
 # tests, and eventually get back to the standard level.
-MIN_COVERAGE := 30
+MIN_COVERAGE := 75
```

Comment:
I think we can just actually delete the `MIN_COVERAGE` from the makefile since the default defined in `subtrees/z_quantum_actions/variables.mk` is 75.
```suggestion
```